ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -11,6 +11,10 @@ base_images:
     name: cincinnati-graph-data
     namespace: cincinnati-ci-public
     tag: stable
+  operator-index:
+    name: redhat-operator-index
+    namespace: ci
+    tag: v4.10
```

Comment:
Why 4.10?

Reply:
The index image doesn't support `SQLite` since OCP 4.11, so we need to use 4.10 here.

Reply:
Yeah, thanks! @shellyyang1989 let's have a try!

Reply:
Thank you Petr and Jian. Removed the deprecated items.